### PR TITLE
NuoCD wrapper for python

### PR DESCRIPTION
**Pull request by vegichan**
Create a new wrapper for python that detects and picks the best available version of Python.
- Python2 only

### CODE REVIEW

File: bin/nuocd [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+die () { echo "$*"; exit 1; }
+
+_python=${PYTHON}
+if test -z "$_python"; then
+  _python=$(command -v python2.7) || _python=$(command -v python2) || _python=$(command -v python)  \
+  || die "Python must be installed"
+fi
+
+echo "Found $_python" >&2
```
Comment (adriansuarez) [inline bin/nuocd:16]: Is this for debugging, or do we want to emit this every time we run nuocd? Perhaps we should have a more descriptive message like `Using Python executable $_python`.

File: bin/nuocd [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```
Comment (adriansuarez) [inline bin/nuocd:1]: Maybe we should call this something else, since it actually is not specific to `nuocd`, e.g. `python-wrapper`.

**Review by adriansuarez — APPROVED** [review]
Looks good.

**Review by madscientist — APPROVED** [review]
Looks good.

**Pull request merged**